app/javascript/components/close_contact/CloseContact.js

```diff
@@ -265,6 +311,11 @@ const schema = yup.object().shape({
     .email('Please enter a valid email.')
     .max(200, 'Max length exceeded, please limit to 200 characters.')
     .nullable(),
+  last_date_of_exposure: yup.date('Date must correspond to the "mm/dd/yyyy" format.').nullable(),
```

Comment:
does the PO team want a min/max date?

Reply:
I asked and the date can be between Jan 1 2020 and 30 days into the future. Updated in the latest commit.